zavod/zavod/helpers/sanctions.py

```diff
@@ -30,4 +43,19 @@ def make_sanction(
         sanction.add("country", dataset.publisher.country)
     sanction.add("authority", dataset.publisher.name)
     sanction.add("sourceUrl", dataset.url)
+
+    if program is not None:
+        sanction.set("program", program)
+        program_id = context.lookup_value("sanction", program)
+        if program_id is None:
+            context.log.warn(f"Program key '{program}' not found.")
+    if start_date is not None:
+        h.apply_date(sanction, "startDate", start_date)
+    if end_date is not None:
+        h.apply_date(sanction, "endDate", end_date)
+        iso_end_date = h.extract_date(context.dataset, end_date)[0]
+        end_date_obj = datetime.strptime(iso_end_date[:10], "%Y-%m-%d")
+        is_active = end_date_obj >= datetime.today()
```

Comment:
Can we compare with `settings.RUN_TIME` not with `datetime.today()`? They'll not always be the same, and there's lots of timezone trouble in this code. 

FWIW it should be OK to compare ISO timestamps instead of parsing the date. And maybe instead of using `h.extract_date` here we can use `max(sanction.get('endDate'))` ?